### ISSUE TITLE
No text anywhere in application 

### ISSUE THREAD

**Issue by martinevans2** [DeedleFake/trayscale#156]
Linux Mint. No text anywhere in application except Preferences and About. Resizing doesn't help. Changing operating system theme doesn't help.

![Screenshot from 2024-08-23 10-40-04](https://github.com/user-attachments/assets/094ee84f-4662-427a-b5a7-dc0867b5b690)

**Comment by DeedleFake** [comment]
Possible duplicate of #105. Can you read through that and see if it's related, please?

**Comment by martinevans2** [comment]
Hi, I looked into that and it certainly seems the same. When I run the
application from the terminal I get the same error message. But I couldn't
see in the thread how to fix it. I went down a rabbit hole of fixing the
underlying problem but I couldn't figure it out. Do you know what the
solution was to #105? Sorry if it's in the thread already. I'm not an
expert at linux and I don't know what is different about Mint vs Arcolinux.
I assume they have the same underlying OS front-end?

**Comment by DeedleFake** [comment]
No, I don't know what the fix was, sorry. Unfortunately, the thread that it links to in turn links to a solution that is in a private repo that I don't have access to.
